### ISSUE TITLE
Provide arm64 download support in install.sh

### ISSUE THREAD

**Issue by Allaman** [krivahtoo/silicon.nvim#34]
Hi, I am really excited about using your plugin, but unfortunately, I guess the ARM artifacts are missing? (I cannot build it myself on my machine)

Running `install.sh` yields (added one line debug output)
<img width="449" alt="image" src="https://user-images.githubusercontent.com/12184268/226476859-d4079815-97b0-4bf5-86a3-6d58485e6917.png">

Thanks!

**Comment by krivahtoo** [comment]
You have to build the library `install.sh build` if you have the rust toolchain setup.

Mac arm64 prebuilds are not available for versions after `v0.3.0`.

**Comment by Allaman** [comment]
That's a pity. Thanks for your reply.

**Comment by Spuxy** [comment]
yep that's bad :( can we know any reason ? silicon by default (the binary) does has problem with this arch ? I was looking at ur changelogs and 0 mention about that :/

**Comment by krivahtoo** [comment]
I am keeping this open until it is fixed

**Comment by theherk** [comment]
> You have to build the library `install.sh build` if you have the rust toolchain setup.
> 
> Mac arm64 prebuilds are not available for versions after `v0.3.0`.

I'm not able to build with nightly.

```
error[E0635]: unknown feature `proc_macro_span_shrink`
  --> /Users/h4s/.cargo/registry/src/index.crates.io-6f17d22bba15001f/proc-macro2-1.0.56/src/lib.rs:92:30
   |
92 |     feature(proc_macro_span, proc_macro_span_shrink)
   |
```

**Comment by krivahtoo** [comment]
This should be fixed on the latest release.

3e51b26
